x/claim/keeper/hooks.go

```diff
@@ -99,6 +99,11 @@ func (h Hooks) AfterEpochEnd(ctx sdk.Context, epochInfo epochstypes.EpochInfo) {
 	h.k.AfterEpochEnd(ctx, epochInfo)
 }
 
+func (h Hooks) AfterUnbondingInitiated(ctx sdk.Context, id uint64) error {
+	// TODO: implement
```

Comment:
what's this for?

Reply:
Think this is a new SDK 47 hook

Reply:
Let's remove the comment, can also remove the hook

Reply:
done